src/containers/Procedures.js

```diff
@@ -1,26 +1,35 @@
 import React, { Component } from 'react';
 import { connect } from 'react-redux';
-import { Header } from '../components/Header';
+import { Redirect } from "react-router-dom"
 import { menuStateChange, redirectFromPage } from '../action/HeaderActions';
 import './App.css'
 
 class Procedures extends Component {
+
     render() {
-        return (
-            <Header />
-        )
+        const isMenuActive = this.props.procedures.isUserMenuActivated,
+        isRedirected = this.props.procedures.isRedirected,
+        navBarActive = this.props.procedures.navBarActive;
+
+        if (isRedirected) {
+            this.props.redirectFromPage(false);
+            return <Redirect push to="/" />;
+            }
+
+        return <p>aas</p>
```

Comment:
It will be better if you should return 'null' and add comment '//TODO something..'